lib/generators/stimulus_reflex/templates/app/javascript/config/cable_ready.js.tt

```diff
@@ -0,0 +1,4 @@
+import consumer from '../channels/consumer'
+import CableReady from 'cable_ready'
+
+CableReady.initialize({ consumer })
```

Comment:
This should live in the CableReady installer

Reply:
My current approach is that while CR will have an installer, it will actually just be the SR installer with a limited subset of steps, and that if you run the CR installer, it will check for the SR gem and if it's present, tell you to just run the SR installer.

So... it's more of an *also*, not an instead of.